### PR TITLE
Added the alerts for the aws cloudtrail data

### PR DESCRIPTION
**Pull request by hardikhdholariya**
Added the alerts for the aws cloudtrail data

### CODE REVIEW

File: cyences_app_for_splunk/default/macros.conf [hunk 1]
```diff
@@ -1365,6 +1365,24 @@ definition = search *
 [cs_aws]
 definition = index IN (aws,summary)
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/macros.conf:1366]: Is `cs_aws_cloudtrail_logs` macro present already?

**Comment by hardikhdholariya** [comment]
This PR does not contain all the alerts. I'll add more alerts in next commits.

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 2]
```diff
@@ -4610,6 +4610,222 @@ search = `cs_aws_account_summary`| dedup name, account_id | table name, account_
...
+action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.param.filter_macro_name = cs_aws_console_login_failed_during_mfa_challenge_filter
+action.cyences_notable_event_action.contributing_events = `cs_aws_cloudtrail_logs` eventName="ConsoleLogin" errorMessage="Failed authentication" additionalEventData.MFAUsed="Yes"
+action.cyences_notable_event_action.system_compromised_search = | stats count by userName
+action.cyences_notable_event_action.system_compromised_drilldown = `cs_aws_cloudtrail_logs` eventName="ConsoleLogin" errorMessage="Failed authentication" additionalEventData.MFAUsed="Yes" userName=$row.userName$
+action.cyences_send_email_action = 1
+action.cyences_notable_event_action.products = AWS
+
+
+[AWS - Created a Policy Version that allows All Resources to be Accessed]
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:4790]: nice alert

**Review by VatsalJagani — APPROVED** [review]
We need to add similar to Dashboard, FYI.

File: cyences_app_for_splunk/default/macros.conf (hunk 1 above)
Comment (hardikhdholariya) [reply cyences_app_for_splunk/default/macros.conf:1366]: Yes, it is present.

**Pull request merged**